modules/base/rendering/pointcloud/renderablepointcloud.cpp

```diff
@@ -318,6 +325,23 @@ namespace {
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
+    constexpr openspace::properties::Property::PropertyInfo OutlineStyleInfo = {
+        "OutlineStyle",
+        "Outline Style",
+        "This setting decides the shape of points that have an outline (round, square, "
+        "or lust a line at the bottom). Note that anything but \"Round\" will lead to "
+        "the point being drawed as squares.",
+        openspace::properties::Property::Visibility::AdvancedUser
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo ApplyColorMapToOutlineInfo = {
+        "ApplyColorMapToOutline",
+        "Apply Color Map to Outline",
+        "If true and outline is enabled, the color map will be applied to the outline "
+        "rather than the point body. Only works if color mapping is enabled.",
```

Comment:
```suggestion
        "If true and the outline is enabled, the color map will be applied to the "
        "outline rather than the point body. Only works if color mapping is enabled.",
```